log4j-core-test/src/main/java/org/apache/logging/log4j/core/test/junit/package-info.java

```diff
@@ -20,7 +20,7 @@
  * @see org.junit.rules.TestRule
  */
 @Export
-@Version("2.23.0")
+@Version("2.23.1")
```

Comment:
```suggestion
@Version("2.25.0")
```

Reply:
Why? It is a `MICRO` change, so I only increased the patch number.

Reply:
Weren't you the one advising that we should use the version number of the release a change will be published with? (If I'm mistaken, you can just ignore my correction, but I'd appreciate an explanation.)

Reply:
For `MINOR` changes yes.
What I would like to convey with the `@Version` annotation is:

- the stability of the API in each package,
- in which minor version was the API last modified (a sort of `max(@since)`).

In this case this package was last modified in `2.23.0`, since then users didn't get any additional functionality, only bug fixes. If they compile their code against `2.23.0`, they will get the same result as using the latest release.